.rubocop_todo.yml

```diff
@@ -26,7 +26,7 @@ Metrics/AbcSize:
 # Offense count: 3
 # Configuration parameters: CountComments, CountAsOne.
 Metrics/ClassLength:
-  Max: 130
+  Max: 135
```

Comment:
Please don't make this change globally as this allows every file to get worse.  Instead, add an inline exception only on the file that requires this.

Reply:
Can you please revert this change?